config/environments/production.rb

```diff
@@ -78,6 +78,7 @@
   # Enable DNS rebinding protection and other `Host` header attacks.
   config.hosts = [
     /release\..*gov.uk?/,
+    /^release?/,
```

Comment:
I've just updated a bunch of ours, on Bruce's suggestion - to protect against `foogov.uk` etc. Would suggest:

```suggestion
    /release\..*\.gov.uk$/,
    /^release$/,
```

Reply:
oh yeah! Whoops